README.md

```diff
@@ -20,16 +20,15 @@ This project is made possible by a [community of contributors](https://github.co
 
 ```bash
 npm install --save ts-json-schema-generator
-./node_modules/.bin/ts-json-schema-generator \
-    --path 'my/project/**.*.ts' \
-    --type 'My.Type.Full.Name' \
-    --expose 'export' \
-    --jsDoc 'extended'
+./node_modules/.bin/ts-json-schema-generator --path 'my/project/**.*.ts'
```

Comment:
The typical usage will be to provide a type so let's leave that here in the docs.

Reply:
> The typical usage will be to provide a type so let's leave that here in the docs.

Agreed.